package.json

```diff
@@ -23,7 +23,6 @@
     "lint:files": "ember adde-lint --file-names"
   },
   "dependencies": {
-    "@ember-decorators/argument": "^0.8.16",
     "@ember-decorators/babel-transforms": "^0.1.1",
```

Comment:
what about this one?

Reply:
I think we need the babel transforms, since we were using classes still, but not sure.